django_project/core/models/collection/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from core.models.metadata.models import Metadata
+
+
+class CollectionType(Metadata):
+    name = models.CharField(max_length=128)  # 'farmed', 'wild-caught', etc.
+
+    def __str__(self):
+        return self.name
+
+
+class CollectionMethod(Metadata):
+    name = models.CharField(max_length=128)  # 'gillnets', 'longline', 'recirculating tanks', etc.
```

Comment:
Is a max length required by the backend DB? Is that a performance optimization?

Reply:
Required for every DB except Postgres https://docs.djangoproject.com/en/5.0/ref/models/fields/#django.db.models.CharField.max_length

I haven't set up Postgres yet, so this is using the Django default of SQLite. I was planning to migrate it later, but if you'd recommend avoiding length constraints I could investigate setting it up sooner